common/src/main/java/com/wynntils/models/territories/GuildAttackTimerModel.java

```diff
@@ -116,7 +118,22 @@ void processChanges(ScoreboardSegment segment) {
             }
         }
 
+        // FIXME: This model should use chat messages to parse new wars,
+        //        and only fall back to scoreboard on first join.
```

Comment:
To be handled by `AttackTimerOverlayFeature (Improved handling for territories cut off scoreboard, either vertically, or horizontally)`